src/Kiota.Builder/Writers/CSharp/CodeClassDeclarationWriter.cs

```diff
@@ -8,13 +8,15 @@
 namespace Kiota.Builder.Writers.CSharp;
 public class CodeClassDeclarationWriter : BaseElementWriter<ClassDeclaration, CSharpConventionService>
 {
+    public static string AutoGenerationHeader => "// <auto-generated/>";
```

Comment:
can you add a validation in the unit test to guard against regressions please?

Reply:
Added UT's for class and enum writer.